project/FrgaalJavaCompiler.scala

```diff
@@ -112,7 +112,7 @@ object FrgaalJavaCompiler {
     def checkTarget(x : Any) = {
       val p = asPath(x)
       val namesCheck = for (i <- 0 until p.getNameCount)
-        yield "target".equals(p.getName(i).toString())
+        yield "target".equals(p.getName(i).toString()) || p.getName(i).toString().endsWith("-windows") || p.getName(i).toString().endsWith("-unix")
```

Comment:
Why do we need those now?

Reply:
I needed this to [Make sure logger-service can be debugged in Enso4Igv modules](https://github.com/enso-org/enso/pull/5684/commits/70e3ab0844789c26610722d2d63ab3635b3a0731). A bit unrelated, but I was trying to investigate #5683 and I cannot do that without being able to debug `logger-service`.